cli/cli-test-runner.ts

```diff
@@ -8,10 +11,54 @@ export class CliTestRunner {
       }
    }
 
-   public run(testSet: TestSet, timeout?: number) {
+   public static create(): CliTestRunner {
+      const outputStream = new TestOutputStream();
+      const testRunner = new TestRunner(outputStream);
+      return new CliTestRunner(testRunner);
+   }
+
+   public run(userArguments: AlsatianCliOptions) {
+
+      // if version has been requested then output the current version and exit
+      if (userArguments.versionRequested) {
+         process.stdout.write("alsatian version " + Package.version);
+         return;
+      }
+
+      // if help has been requested then output the current version and exit
```

Comment:
haha that was supposed to read the CLI options, their descriptions and the current version... dear me :P #copyandpastemonster I'll fix that tonight - good spot